### PR TITLE
chore(website): Product Updates #6

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

**Comment by jsonpr** [comment]
Do we want to add in partner shoutouts?

- [Apache Arrow] (https://arrow.apache.org/blog/2023/05/04/adopting-apache-arrow-at-cloudquery/)
- [Neo4J Live] (https://www.youtube.com/watch?v=o-thc4SVDFg)
- [SumoLogic] (https://www.sumologic.com/blog/power-community-driven-cloud-security-cspm/)

File: website/pages/blog/product-updates-6.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here!
+
+## Apache Arrow
+
+We were heads down as we were adopting Apache Arrow as our in memory [data format](/blog/adopting-apache-arrow-at-cloudquery)!
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-6.mdx:18]:
```suggestion
We were heads down adopting Apache Arrow as our [in-memory data format](/blog/adopting-apache-arrow-at-cloudquery)!
```

File: website/pages/blog/product-updates-6.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* [Coinpaprika](https://github.com/coinpaprika/cq-source-coinpaprika)
+* [Onfleet](https://github.com/onfleet/cq-source-onfleet)
+* [CrowdStrike](https://github.com/justmiles/cq-source-crowdstrike)
+* [Sharepoint](https://github.com/koltyakov/cq-source-sharepoint)
+
+New Official plugins:
+
+* [OracleDB](/docs/plugins/sources/oracledb/overview)
+* [Firestore](/docs/plugins/sources/firestore/overview) Thanks to to [@BubbaJoe](https://github.com/BubbaJoe) for initial contribution!
```
Comment (hermanschaaf) [inline website/pages/blog/product-updates-6.mdx:40]:
```suggestion
* [Firestore](/docs/plugins/sources/firestore/overview) Thanks to [@BubbaJoe](https://github.com/BubbaJoe) for initial contribution!
```

**Review by hermanschaaf — APPROVED** [review]
🚀

**Pull request merged**